pkg/datapath/linux/requirements.go

```diff
@@ -176,7 +176,12 @@ func CheckMinRequirements() {
 	if !option.Config.DryMode {
 		probeManager := probes.NewProbeManager()
 		if err := probeManager.SystemConfigProbes(); err != nil {
-			log.WithError(err).Warning("BPF system config check: NOT OK.")
+			errMsg := "BPF system config check: NOT OK."
+			if err == probes.ErrKernelConfigNotFound {
+				log.WithError(err).Info(errMsg)
+			} else {
+				log.WithError(err).Fatal(errMsg)
```

Comment:
Do we really want to fatal here?

Reply:
I changed it to `Fatal` from `Error` as @mrostecki suggested
![image](https://user-images.githubusercontent.com/21292343/84372572-ef1b3c00-abf8-11ea-8201-7bf19ebc4330.png)

Reply:
Yes, I think that `Fatal` is appropriate here. If the kernel config is available and we can check it, we should mark the BPF check as failed, right? And for the `probeManager.CreateHeadersFile()` check below we also use `Fatal`.

Reply:
My reasoning was that even if cilium-agent cannot detect the config, it can still function properly.

Reply:
^^^

IIRC Core OS was the distro in the past we had the most trouble with in this area. Maybe a more recent example would be something like Flatcar Linux. If we know it runs on that distro fine, then it should likely work on any distro.

Reply:
@mrostecki @fristonio Let's keep it as Warn message instead.